src/components/AssetIcon.tsx

```diff
@@ -118,14 +117,48 @@ export const AssetIcon: React.FC<AssetIconProps> = ({
 
   // For Soroban custom tokens, use the Soroban logo
   if (token.type === AssetTypeWithCustomToken.CUSTOM_TOKEN) {
+    const tokenIdentifier = getTokenIdentifier(token);
```

Comment:
I don't think our current implementation of `getTokenIdentifier` will work with custom tokens